### PR TITLE
Coding standards

### LINKED ISSUE #42: Coding Standards
Issue for reference

### PR DESCRIPTION
**Pull request by DannyWPWS**
Fixes #42

### CODE REVIEW

File: core/class-extension-core.php [hunk 1]
```diff
@@ -101,91 +111,98 @@ public function load_plugin_textdomain() {
...
 					/* Notify the user that the activity log is not installed */
-				?>
-                <div class="notice notice-success is-dismissible wsal-installer-notice">
+					?>
+				<div class="notice notice-success is-dismissible wsal-installer-notice">
 					<?php
 					printf(
 						'<p>%1$s &nbsp;&nbsp;<button class="install-wsal button button-primary" data-plugin-slug="wp-security-audit-log/wp-security-audit-log.php" data-plugin-download-url="%2$s" data-plugins-network="%4$s" data-nonce="%3$s">%5$s</button><span class="spinner" style="display: none; visibility: visible; float: none; margin: 0 0 0 8px;"></span></p>',
 						sprintf(
+							/* translators: %s: pluign name. */
```
Comment (martinkrcho) [inline core/class-extension-core.php:181]:
```suggestion
							/* translators: %s: plugin name. */
```

File: core/class-plugin-installer.php [hunk 2]
```diff
@@ -96,7 +99,7 @@ public function install_plugin( $plugin_zip = '' ) {
 			}
 			// clear the cache so we're using fresh data.
 			wp_cache_flush();
-			$upgrader       = new \Plugin_Upgrader();
+			$upgrader       = new \Plugin_Upgrader;
```
Comment (martinkrcho) [inline core/class-plugin-installer.php]: Let's keep the brackets here.
```suggestion
			$upgrader       = new \Plugin_Upgrader();
```

File: core/class-extension-core.php [hunk 3]
```diff
@@ -61,11 +70,12 @@ public function __construct( $extension_main_file_path = '', $text_domain = '' )
 				$this->custom_sensor_path    = ( isset( $extension_main_file_path['custom_sensor_path'] ) ) ? $extension_main_file_path['custom_sensor_path'] : '';
 				$this->extension_plugin_name = '';
 			}
+
 			// If we dont have array, then continue with the as normal.
```
Comment (martinkrcho) [inline core/class-extension-core.php]:
```suggestion
			// If we don't have an array, then continue with the as normal.
```

File: core/class-extension-core.php [hunk 4]
```diff
@@ -101,91 +111,98 @@ public function load_plugin_textdomain() {
...
-			$freeActivated = false;
-			$premiumActivated = false;
+			$free_installed    = false;
+			$premium_installed = false;
+			$free_activated    = false;
+			$premium_activated = false;
 
-			/* Starting checks */
+			// Starting checks.
 			/* Is there free version installed */
```
Comment (martinkrcho) [inline core/class-extension-core.php]:
```suggestion
			// Is there free version installed?
```

File: core/class-extension-core.php [hunk 5]
```diff
@@ -101,91 +111,98 @@ public function load_plugin_textdomain() {
...
-			if ( $plugin_installer->is_plugin_installed( 'wp-security-audit-log/wp-security-audit-log.php' )) {
-				$freeInstalled = true;
+			if ( $plugin_installer->is_plugin_installed( 'wp-security-audit-log/wp-security-audit-log.php' ) ) {
+				$free_installed = true;
 			}
 			/* Is there premium version installed */
```
Comment (martinkrcho) [inline core/class-extension-core.php]:
```suggestion
			// Is there premium version installed?
```

File: core/class-extension-core.php [hunk 6]
```diff
@@ -101,91 +111,98 @@ public function load_plugin_textdomain() {
...
+			// End checks.
 
-			if ( $freeInstalled || $premiumInstalled ) {
-				/* We have plugin installed */
-				/* Is free version activated */
-				if (is_plugin_active( 'wp-security-audit-log/wp-security-audit-log.php' )) {
-					$freeActivated = true;
+			if ( $free_installed || $premium_installed ) {
+				// We have plugin installed.
+				// Is free version activate.
```
Comment (martinkrcho) [inline core/class-extension-core.php]:
```suggestion
				// Is free version active?
```

File: core/class-extension-core.php [hunk 7]
```diff
@@ -101,91 +111,98 @@ public function load_plugin_textdomain() {
...
+				if ( is_plugin_active( 'wp-security-audit-log/wp-security-audit-log.php' ) ) {
+					$free_activated = true;
 				}
-				/* Is premium version activated */
-				if (is_plugin_active( 'wp-security-audit-log-premium/wp-security-audit-log.php' )) {
-					$premiumActivated = true;
+				// Is premium version activated.
```
Comment (martinkrcho) [inline core/class-extension-core.php]:
```suggestion
				// Is premium version active?
```

File: core/class-extension-core.php [hunk 8]
```diff
@@ -101,91 +111,98 @@ public function load_plugin_textdomain() {
...
+				if ( is_plugin_active( 'wp-security-audit-log-premium/wp-security-audit-log.php' ) ) {
+					$premium_activated = true;
 				}
 
-				if ( $freeActivated || $premiumActivated ) {
+				if ( $free_activated || $premium_activated ) {
 					/* There is installed and activated plugin - bounce */
```
Comment (martinkrcho) [inline core/class-extension-core.php]:
```suggestion
					// There is installed and activated plugin - bounce.
```

File: core/class-extension-core.php [hunk 9]
```diff
@@ -101,91 +111,98 @@ public function load_plugin_textdomain() {
...
 					return;
 				} else {
 
-					if ( ! self::$adminNoticeAlreadyShown ) {
-					/* Notify the user that the activity log is not active */
-					?>
+					if ( ! self::$admin_notice_already_shown ) {
+						/* Notify the user that the activity log is not active */
```
Comment (martinkrcho) [inline core/class-extension-core.php]:
```suggestion
						// Notify the user that the WP Activity Log plugin is not active.
```

File: core/class-extension-core.php [hunk 10]
```diff
@@ -101,91 +111,98 @@ public function load_plugin_textdomain() {
...
-					<?php
-						self::$adminNoticeAlreadyShown = true;
+						<?php
+						self::$admin_notice_already_shown = true;
 					}
 				}
 			} elseif ( ! class_exists( 'WpSecurityAuditLog' ) ) {
-				if ( ! self::$adminNoticeAlreadyShown ) {
+				if ( ! self::$admin_notice_already_shown ) {
 					/* Notify the user that the activity log is not installed */
```
Comment (martinkrcho) [inline core/class-extension-core.php]:
```suggestion
					// Notify the user that the WP Activity Log is not installed.
```

**Review by martinkrcho — CHANGES_REQUESTED** [review]
I suggested lots of little changes, @DannyWPWS . Sorry for being a bit pedantic, but I'd like to make sure we do this only once this time round :)

File: core/class-plugin-installer.php (hunk 2 above)
Comment (DannyWPWS) [reply core/class-plugin-installer.php]: Ok iv disabled warnings for this

**Review by martinkrcho — APPROVED** [review]
Everything is looking much  better now. Thanks for sticking with it 👍🏼

**Pull request merged**